core/src/avm2/globals/flash/filters/ShaderFilter.as

```diff
@@ -15,6 +15,11 @@ package flash.filters {
         public function set shader(value:Shader):void {
             this._shader = value;
         }
+
+        public var bottomExtension:int = 0;
+        public var leftExtension:int = 0;
+        public var rightExtension:int = 0;
+        public var topExtension:int = 0;
```

Comment:
AFAIK FP has constraints on these and uses a getter/setter for them. While figuring out the constraints might take some reverse-engineering (since we can't look at the playerglobal code), for consistency and later use these should probably be made into getters and setters.